nuitka/nodes/NodeMakingHelpers.py

```diff
@@ -47,12 +46,13 @@ def makeRaiseExceptionReplacementExpression(expression, exception_type,
                                             exception_value):
     from .ExceptionNodes import ExpressionRaiseException
     from .BuiltinRefNodes import ExpressionBuiltinExceptionRef
+    from nuitka import Options
 
     source_ref = expression.getSourceReference()
 
     assert type(exception_type) is str
 
-    if shallWarnImplicitRaises():
+    if Options.shallWarnImplicitRaises():
```

Comment:
The reason I have done "from" imports is also performance, not sure, should let go of that though, premature optimization likely. I do however fancy seeing what is used from a module right at the top.

Reply:
Well, first I'd say we complete this refactoring, then refactor it even more, more, ... And when the whole system becomes a little bit more sanitized then and just then we can optimize it. Haven't measure how much would be the overhead of having this local imports. In my projects I don't favour local imports and I do favour clean and tidy subsystems with a minimal and clean set of global imports. In fact, I do favour as little imports as the subsystem requires so if somebody just look the imports he won't even read the code and have a grasp about the high level interface of the code. One thing I've noticed on many modules is the top level files contains a lot of imports where you can't really say easily what's the interconection between modules/subsystems